### PR TITLE
4.2.0 alpha release notes

### PR DESCRIPTION
**Pull request by sandrobonazzola**
Changes proposed in this pull request:

- 4.2.0 alpha release notes

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @sandrobonazzola

This pull request needs review by: @JohnMarksRH @mykaul

### CODE REVIEW

File: source/release/4.2.0/index.html.md [hunk 1]
```diff
@@ -0,0 +1,710 @@
...
+- oVirt 4.2.0 uses **PostgresSQL 9.5** as its database, for improved performance.
+
+- Support is now restored for **Gluster ISO domains**, without the need for NFS which was previously disabled by Gluster. 
+
+- **Affinity labels** create a positive affinity between the hosts and VMs to which they are applied. The new Affinity Labels sub tab for clusters, hosts, and VMs in the Administration Portal provides a table view of labels associated with the currently selected entity, as well as the option to add, edit, and delete them. Additionally, existing labels can be added and/or removed from VMs and hosts in their respective new/edit popup dialogs.
+
+To learn about features introduced before 4.2.0, see the [release notes for previous versions](/documentation/#previous-release-notes).
+
+
+## Install / Upgrade from previous versions
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: I am not clear on the structure / relationship between these three headings:
## Install / Upgrade from previous versions
### Fedora / CentOS / RHEL
## ALPHA RELEASE

File: source/release/4.2.0/index.html.md [hunk 2]
```diff
@@ -0,0 +1,710 @@
...
+
+
+In order to install it on a clean system, you need to install
+
+
+`# yum install `[`http://resources.ovirt.org/pub/yum-repo/ovirt-release42-pre.rpm`](http://resources.ovirt.org/pub/yum-repo/ovirt-release42-pre.rpm)
+
+
+and then follow our
+[Installation Guide](http://www.ovirt.org/documentation/install-guide/Installation_Guide/)
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: Add full stop.

File: source/release/4.2.0/index.html.md [hunk 3]
```diff
@@ -0,0 +1,710 @@
...
+
+
+
+### oVirt Hosted Engine
+
+If you're going to install oVirt as Hosted Engine on a clean system please
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: change to "as a Hosted Engine"

File: source/release/4.2.0/index.html.md [hunk 4]
```diff
@@ -0,0 +1,710 @@
...
+If you're upgrading an existing Hosted Engine setup, please follow
+[Hosted_Engine_Howto#Upgrade_Hosted_Engine](/documentation/how-to/hosted-engine/#upgrade-hosted-engine)
+guide or the corresponding section within the
+[Upgrade Guide](/documentation/upgrade-guide/upgrade-guide/).
+
+### EPEL
+
+TL;DR Don't enable all of EPEL on oVirt machines.
+
+The ovirt-release package enables the epel repositories and includes several
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: Capitalize EPEL

File: source/release/4.2.0/index.html.md [hunk 5]
```diff
@@ -0,0 +1,710 @@
...
+specific packages that are required from there. It also enables and uses
+the CentOS SIG repos, for other packages.
+
+If you want to use other packages from EPEL, you should make sure to
+use `includepkgs` and add only those you need avoiding to override
+packages from other repos.
+
+### Deprecated Functionality
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: Suggest to move this section BELOW "enhancements"

File: source/release/4.2.0/index.html.md [hunk 6]
```diff
@@ -0,0 +1,710 @@
...
+
+ - [BZ 1464527](https://bugzilla.redhat.com/1464527) <b>[centos-qemu-ev-release] GPG key not found</b><br>
+ - [BZ 1488189](https://bugzilla.redhat.com/1488189) <b>Enable centos-release-scl-rh on el7 x86_64</b><br>
+ - [BZ 1443965](https://bugzilla.redhat.com/1443965) <b>Libvirt is disabled on RHVH host</b><br>
+
+#### oVirt Engine Appliance
+
+ - [BZ 1406493](https://bugzilla.redhat.com/1406493) <b>Use livemedia-creator instead of image-tools</b><br>
+
+### No Doc Update
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: What does "no doc update" mean?

File: source/release/4.2.0/index.html.md [hunk 7]
```diff
@@ -0,0 +1,710 @@
...
+
+#### oVirt Engine Appliance
+
+ - [BZ 1463853](https://bugzilla.redhat.com/1463853) <b>[RFE] RHV-M appliance should meet NIST 800-53 partitioning requirements</b><br>Feature: <br>Modified disk partitioning scheme for RHVM-Appliance from 2 primary partitions ("/" and swap) to the one specified by NIST.<br><br>Reason: <br>Meet with NIST partitioning requirements<br><br>Result: <br>Updated disk partitions to be as follows:<br><br>/boot 1G (primary)<br>/home 1G (lvm)<br>/tmp 2G (lvm)<br>/var 20G (lvm)<br>/var/log 10G (lvm)<br>/var/log/audit 1G (lvm)<br>swap 8G (lvm)<br>/ 6G (primary)
+ - [BZ 1422982](https://bugzilla.redhat.com/1422982) <b>[RFE] add swap to rhevm-appliance</b><br>Feature: rhvm-appliance now includes a swap partition as part of the base install.<br><br>Reason: In some cases, the default memory allotment for rhvm-appliance may not be large enough to include support for user additions. Rather than increasing the memory for everyone, swap offers users more flexibility.<br><br>Result: There is a swap partition as part of the default build of rhvm-appliance
+
+### Known Issue
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: In what sense is it "known issue"?

File: source/release/4.2.0/index.html.md [hunk 8]
```diff
@@ -0,0 +1,710 @@
...
+
+ - [BZ 1446480](https://bugzilla.redhat.com/1446480) <b>[RFE]  add "Uptime" Column to oVirt Metrics Store</b><br>Feature: <br>This patch loads the uptime collectd plugin that reports the system uptime.<br><br>Reason: <br><br>Result:
+
+#### oVirt Hosted Engine Setup
+
+ - [BZ 1461251](https://bugzilla.redhat.com/1461251) <b>[bug] hosted-engine yum repo required, but rpm-based install optional</b><br>Feature: <br>Provide path to Appliance OVF<br><br>Reason: <br>This feature is supported according to RedHat customer portal, but dropped in the past for some reason<br><br>Result: <br>User can provide path to appliance OVA instead of installing appliance RPM
+ - [BZ 1233127](https://bugzilla.redhat.com/1233127) <b>[RFE] Warn when bad bond modes are used for the bond which is used for the initial VM network</b><br>
+ - [BZ 1429537](https://bugzilla.redhat.com/1429537) <b>[RFE] Rebase on gluster-3.10</b><br>oVirt Hosted Engine now requires GlusterFS 3.10 instead of 3.8.
+
+#### oVirt Log collector
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: Collector

File: source/release/4.2.0/index.html.md [hunk 9]
```diff
@@ -0,0 +1,710 @@
...
+ - [BZ 1454536](https://bugzilla.redhat.com/1454536) <b>HostedEngine setup fails if RHV-H timezone < UTC set during installation</b><br>Cause: RHV-H generates vdsm certificates at the time of first boot.<br><br>Consequence: If the system clock is not set correctly at install time, chrony or ntpd may resynchronize the clock after the vdsm certificate is generated, leading to a certificate which is not valid yet if the appropriate timezone is behind UTC.<br><br>Workaround (if any): Set the system clock appropriately at install time. imgbased-configure-vdsm will now start after chronyd/ntpd and wait 2 seconds for the clock to sync, but this is not a guarantee.
+
+
+### Rebase: Enhancementss Only
+
+#### oVirt Release Package
+
+ - [BZ 1429541](https://bugzilla.redhat.com/1429541) <b>Rebase on Gluster 3.10</b><br>oVirt release now enables CentOS Storage SIG Gluster 3.10 repository. See Gluster 3.10 release notes for more information  at https://gluster.readthedocs.io/en/latest/release-notes/3.10.0/
+
+### Release Note
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: In what sense is it "release note"?

File: source/release/4.2.0/index.html.md [hunk 10]
```diff
@@ -0,0 +1,710 @@
...
+
+## Bug fixes
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: I think it should be ### Bug Fixes

File: source/release/4.2.0/index.html.md [hunk 11]
```diff
@@ -0,0 +1,710 @@
...
+
+### OTOPI
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: ####?

File: source/release/4.2.0/index.html.md [hunk 12]
```diff
@@ -0,0 +1,710 @@
...
+
+ - [BZ 1328764](https://bugzilla.redhat.com/1328764) <b>Setup does not dump keys that are initialized to None</b><br>
+
+### VDSM
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: ####?

File: source/release/4.2.0/index.html.md [hunk 13]
```diff
@@ -0,0 +1,710 @@
...
+ - [BZ 1487560](https://bugzilla.redhat.com/1487560) <b>[iSCSI] ovirt-hosted-engine-setup fails if none of the discovered target is associated to the accessed portal</b><br>
+ - [BZ 1449565](https://bugzilla.redhat.com/1449565) <b>ovirt-hosted-engine-setup has leftover mounts</b><br>
+ - [BZ 1286568](https://bugzilla.redhat.com/1286568) <b>HA Agent and Broker logs have incorrect permissions/ownership</b><br>
+ - [BZ 1481680](https://bugzilla.redhat.com/1481680) <b>hosted-engine --upgrade-appliance fails with KeyError: 'stopped' if the metadata area contains references to 3.5 decommissioned hosts</b><br>
+
+### oVirt ISO Uploader
+
+ - [BZ 1437799](https://bugzilla.redhat.com/1437799) <b>[RFE] - ovirt-iso-uploader should work with Gluster without NFS enabled</b><br>
+
+### oVirt Log collector
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: ####

Comment (JohnMarksRH) [reply source/release/4.2.0/index.html.md]: ####

File: source/release/4.2.0/index.html.md (hunk 11 above)
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: should be ####

File: source/release/4.2.0/index.html.md (hunk 12 above)
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: Should be ####

File: source/release/4.2.0/index.html.md [hunk 14]
```diff
@@ -0,0 +1,710 @@
...
+### VDSM
+
+ - [BZ 1446492](https://bugzilla.redhat.com/1446492) <b>Storage domain in 4.1 RHV will go offline if LVM metadata was restored manually</b><br>
+ - [BZ 1486543](https://bugzilla.redhat.com/1486543) <b>Migration leads to VM running on 2 Hosts</b><br>
+ - [BZ 1358717](https://bugzilla.redhat.com/1358717) <b>Export of vm with thin provision disk from NFS Data domain and Import to Block Data domain makes virtual and Actual size of disk same.</b><br>
+ - [BZ 1326799](https://bugzilla.redhat.com/1326799) <b>[CodeChange] Remove workaround for selinux.restorecon for disabled selinux</b><br>
+ - [BZ 1428851](https://bugzilla.redhat.com/1428851) <b>RDMA Glusterfs does not recognized as mounted</b><br>
+ - [BZ 1412455](https://bugzilla.redhat.com/1412455) <b>[Bug] Gluster brick created with RHEV manager is overallocated</b><br>
+
+### oVirt image transfer daemon and proxy
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: Should be ####

File: source/release/4.2.0/index.html.md [hunk 15]
```diff
@@ -0,0 +1,710 @@
...
+
+ - [BZ 1446094](https://bugzilla.redhat.com/1446094) <b>ovirt-imageio setup doesn't consume --reconfigure-optional-components on engine-setup</b><br>
+
+### oVirt Engine
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: Should be ####

File: source/release/4.2.0/index.html.md [hunk 16]
```diff
@@ -0,0 +1,710 @@
...
+ - [BZ 1366960](https://bugzilla.redhat.com/1366960) <b>Disk element under storagedomains/{domain-id}/disks;unregistered should have 'register' Action</b><br>
+ - [BZ 1484392](https://bugzilla.redhat.com/1484392) <b>Unable to Import a VM linked with its template to RHV 4.1 from export domain imported of RHEV 3.5</b><br>
+ - [BZ 1425879](https://bugzilla.redhat.com/1425879) <b>NFS storage domain path with a trailing slash should be accepted</b><br>
+ - [BZ 1333409](https://bugzilla.redhat.com/1333409) <b>Inconsistent behavior when creating VM from template that pinned to hosts via REST and via UI</b><br>
+ - [BZ 1452139](https://bugzilla.redhat.com/1452139) <b>No validation on the VM disks attachments when taking a snapshot</b><br>
+ - [BZ 1430865](https://bugzilla.redhat.com/1430865) <b>ERROR: duplicate key value violates unique constraint "pk_unregistered_disks_to_vms"</b><br>
+ - [BZ 1406858](https://bugzilla.redhat.com/1406858) <b>Engine setup fails on F25 - m2crypto fails to load the ca.pem</b><br>
+ - [BZ 1315771](https://bugzilla.redhat.com/1315771) <b>Virtual Machines' Disks & Snapshots in Resources in UserPortal is not expanded correctly</b><br>
+
+### ovirt-engine-dwh
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: Should be ####

File: source/release/4.2.0/index.html.md [hunk 17]
```diff
@@ -0,0 +1,710 @@
...
+
+ - [BZ 1167903](https://bugzilla.redhat.com/1167903) <b>versionlock.list is not filtered if engine is not installed</b><br>
+ - [BZ 1263785](https://bugzilla.redhat.com/1263785) <b>Remove constants duplication in ovirt-engine-dwh and ovirt-engine-setup</b><br>
+
+### oVirt Hosted Engine HA
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: Should be ####

File: source/release/4.2.0/index.html.md [hunk 18]
```diff
@@ -0,0 +1,710 @@
...
+
+ - [BZ 1337914](https://bugzilla.redhat.com/1337914) <b>Hosted engine issues too many lvm operations</b><br>
+ - [BZ 1286568](https://bugzilla.redhat.com/1286568) <b>HA Agent and Broker logs have incorrect permissions/ownership</b><br>
+
+### oVirt Hosted Engine Setup
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: Should be ####

File: source/release/4.2.0/index.html.md [hunk 19]
```diff
@@ -0,0 +1,710 @@
...
+
+ - [BZ 1464461](https://bugzilla.redhat.com/1464461) <b>hosted-engine --upgrade-appliance reports unsupported upgrade path</b><br>
+ - [BZ 1466234](https://bugzilla.redhat.com/1466234) <b>Hosted Engine upgrade from 3.6 to 4.0 will fail if the NFS is exported with root_squash</b><br>
+ - [BZ 1449557](https://bugzilla.redhat.com/1449557) <b>ovirt-hosted-engine-setup installs an older HE appliance and then upgrade to latest HE image (currently RHV-4.1)</b><br>
+ - [BZ 1487560](https://bugzilla.redhat.com/1487560) <b>[iSCSI] ovirt-hosted-engine-setup fails if none of the discovered target is associated to the accessed portal</b><br>
+ - [BZ 1449565](https://bugzilla.redhat.com/1449565) <b>ovirt-hosted-engine-setup has leftover mounts</b><br>
+ - [BZ 1286568](https://bugzilla.redhat.com/1286568) <b>HA Agent and Broker logs have incorrect permissions/ownership</b><br>
+ - [BZ 1481680](https://bugzilla.redhat.com/1481680) <b>hosted-engine --upgrade-appliance fails with KeyError: 'stopped' if the metadata area contains references to 3.5 decommissioned hosts</b><br>
+
+### oVirt ISO Uploader
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: Should be ####

File: source/release/4.2.0/index.html.md (hunk 13 above)
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: Should be ####

File: source/release/4.2.0/index.html.md [hunk 20]
```diff
@@ -0,0 +1,710 @@
...
+ - [BZ 1469104](https://bugzilla.redhat.com/1469104) <b>Remove host statistics collected from vdsm except for storage</b><br>
+ - [BZ 1469119](https://bugzilla.redhat.com/1469119) <b>Disable collectd virt plugin</b><br>
+ - [BZ 1468894](https://bugzilla.redhat.com/1468894) <b>warning message appears when running the metrics setup script regarding jinja2 templating</b><br>
+
+#### oVirt Hosted Engine Setup
+
+ - [BZ 1356425](https://bugzilla.redhat.com/1356425) <b>[TEXT] 'hosted-engine --vm-start' said it destroyed the VM</b><br>
+ - [BZ 1348225](https://bugzilla.redhat.com/1348225) <b>hosted-engine --deploy sorts iscsi luns incorrectly</b><br>
+
+#### oVirt Log collector
```
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: Collector

File: source/release/4.2.0/index.html.md (hunk 6 above)
Comment (JohnMarksRH) [inline source/release/4.2.0/index.html.md]: What does "No Doc Update" mean?

File: source/release/4.2.0/index.html.md [hunk 21]
```diff
@@ -0,0 +1,710 @@
...
+title: oVirt 4.2.0 Release Notes
+category: documentation
+layout: toc
+authors: sandrobonazzola
+---
+
+# oVirt 4.2.0 Release Notes
+
+The oVirt Project is pleased to announce the availability of the 4.2.0
+First Alpha release
```
[the remark this answers is not in the document]
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: There isn't one planned but there's one month before beta and I expect a request to refresh alpha at some point before beta release.

File: source/release/4.2.0/index.html.md [hunk 22]
```diff
@@ -0,0 +1,710 @@
...
+ as of September 21, 2017.
```
[the remark this answers is not in the document]
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: I'll do as soon as I have a final date

File: source/release/4.2.0/index.html.md [hunk 23]
```diff
@@ -0,0 +1,710 @@
...
+
+oVirt is an open source alternative to VMware™ vSphere™, providing an
+awesome KVM management interface for multi-node virtualization.
+This release is available now for Red Hat Enterprise Linux 7.4,
+CentOS Linux 7.4 (or similar).
+
+
+This is pre-release software.
```
[the remark this answers is not in the document]
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: ok

File: source/release/4.2.0/index.html.md [hunk 24]
```diff
@@ -0,0 +1,710 @@
...
+Please take a look at our [community page](/community/) to know how to
```
[the remark this answers is not in the document]
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Updated the template

File: source/release/4.2.0/index.html.md (hunk 23 above)
[the remark this answers is not in the document]
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Updated the template

File: source/release/4.2.0/index.html.md [hunk 25]
```diff
@@ -0,0 +1,710 @@
...
+ask questions and interact with developers and users.
+All issues or bugs should be reported via the
```
[the remark this answers is not in the document]
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Updated the template

File: source/release/4.2.0/index.html.md [hunk 26]
```diff
@@ -0,0 +1,710 @@
...
+For a general overview of oVirt, read the [Quick Start Guide](/documentation/quickstart/quickstart-guide/)
+and visit the [About oVirt](/documentation/introduction/about-ovirt/) page.
+
+For detailed installation instructions, read the [Installation Guide](/documentation/install-guide/Installation_Guide/).
+
+## What's New in 4.2.0?
+
+- The **Administration Portal** has been redesigned from scratch using [Patternfly](http://www.patternfly.org), a widely adopted standard in web application design that promotes consistency and usability across IT applications, through UX patterns and widgets. The result is a cleaner, more intuitive and user-friendly user interface. The old horizontal menu has been replaced by a two-level vertical menu. The system tree is gone, and its functionality has been integrated into the vertical menus.
+
+- An all new **VM Portal** for non-admin users - designed with React-based UI and Patternfly principles - replaces the existing User Portal. Built with performance and ease of use in mind, the VM Portal keeps the Extended view of the User Portal, plus a streamlined functionality.
```
[the remark this answers is not in the document]
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Please go ahead and edit the PR directly

File: source/release/4.2.0/index.html.md (hunk 2 above)
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Added to template

File: source/release/4.2.0/index.html.md (hunk 3 above)
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Added to template

File: source/release/4.2.0/index.html.md (hunk 4 above)
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Updated template

File: source/release/4.2.0/index.html.md (hunk 5 above)
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: I'll try to change the tool for autogeneration setting the requested order.

File: source/release/4.2.0/index.html.md (hunk 8 above)
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Changed in the 4.2.0 conf release

File: source/release/4.2.0/index.html.md (hunk 7 above)
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: In the sense that we know this affect the oVirt release and we haven't fixed it yet but manual action can be done to workaround. Bugs here may have been set with wrong value of doc type.

File: source/release/4.2.0/index.html.md (hunk 10 above)
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Updated the template

File: source/release/4.2.0/index.html.md (hunk 11 above)
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Updated the template

Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Updated the template

File: source/release/4.2.0/index.html.md (hunk 12 above)
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Updated the template

Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Updated the template

File: source/release/4.2.0/index.html.md (hunk 14 above)
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Updated the template

File: source/release/4.2.0/index.html.md (hunk 15 above)
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Updated the template

File: source/release/4.2.0/index.html.md (hunk 16 above)
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Updated the template

File: source/release/4.2.0/index.html.md (hunk 17 above)
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Updated the template

File: source/release/4.2.0/index.html.md (hunk 18 above)
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Updated the template

File: source/release/4.2.0/index.html.md (hunk 19 above)
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Updated the template

File: source/release/4.2.0/index.html.md (hunk 13 above)
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Updated the template

File: source/release/4.2.0/index.html.md (hunk 6 above)
Comment (sandrobonazzola) [reply source/release/4.2.0/index.html.md]: Developer decided that no special documentation is needed

**Pull request merged**